basis/compiler/cfg/stacks/clearing/clearing.factor

```diff
@@ -20,6 +19,6 @@ IN: compiler.cfg.stacks.clearing
     [ [ clearing-replaces ] keep suffix ] with map V{ } concat-as ;
 
 : clear-uninitialized ( cfg -- )
-    [ trace-stack-state ] keep [
+    [ trace-stack-state2 ] keep [
```

Comment:
Is there a better name for this, or is intended to replace the old one and eventually be just called ``trace-stack-state2``?